src/datasources/asset/AssetDataSource.test.ts

```diff
@@ -208,15 +210,39 @@ const assetsResponseMock: AssetsResponse =
     "totalCount": 4
   }
 
+const assetsResponseMockCalibrationForecast: CalibrationForecastResponse = {
+  calibrationSummary: {
+    totalAssets: 10,
+    assetsApproachingCalibration: 6,
+    assetsPastDueCalibration: 4,
+  },
+  calibrationForecast: {
+    columns: [
+      { name: 'Month', values: ['2024-03', '2024-04', '2024-05', '2024-06', '2024-07', '2024-08', '2024-09', '2024-10', '2024-11', '2024-12'] },
```

Comment:
Let's check the date format with TJ. I think we might need to do some date formatting for different 'group by' values.

Reply:
We could also keep it simple and go with the roundtrip format and leave it to the the user